app/src/org/commcare/tasks/DumpTask.java

```diff
@@ -237,7 +236,8 @@ protected Boolean doTaskBackground(String... params) {
                             }
                         }
                     } catch(SessionUnavailableException sue) {
-                        throw sue;

```

Comment:
It makes sure that `SessionUnavailableException` are not caught by the `catch (Exception e)` statement below, and instead handled with the logic I've just moved into the first catch